libraries/SITL/SIM_JSON.h

```diff
@@ -81,6 +81,12 @@ class JSON : public Aircraft {
         Vector3f attitude;
         Quaternion quaternion;
         Vector3f velocity;
+        float rng_1;
```

Comment:
Why not an array?

Reply:
Because I assumed a array wouldn't work with the JSON parser but it does, fixed